daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/crypto/Hash.scala

```diff
@@ -232,6 +232,7 @@ object Hash {
     val MaintainerContractKeyUUID = Purpose(4)
```

Comment:
Also not really about this PR, but since you're here could you fix the typo above "one" -> "once"?

Reply:
Done.